Documentation cleanup

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#23148 Documentation cleanup**

Differential Revision: [D16414202](https://our.internmc.facebook.com/intern/diff/D16414202)

Commit:
Documentation cleanup

Commit:
Update on "Documentation cleanup"

Documentation cleanup

gh-metadata: pytorch pytorch 23148 gh/zafartahirov/14/head

zafartahirov commented:
@pytorchbot retest this please

Commit:
Update on "Documentation cleanup"

Documentation cleanup

gh-metadata: pytorch pytorch 23148 gh/zafartahirov/14/head

soumith commented on torch/nn/quantized/modules/conv.py:
```diff
@@ -14,9 +14,48 @@
+    several quantized input planes.
+
+    For details on input arguments, parameters, and implementation see
+    :class:`~torch.nn.quantized.Conv2d`.
```
you mean `torch.nn.Conv2d` here, but you wrote `quantized` which is circular reference

soumith commented on torch/nn/quantized/modules/conv.py:
```diff
@@ -14,9 +14,48 @@
+
+    .. note::
+        The :attr:`weight` is stored as a packed tensor, and can be accessed
+        using :attr:`_packed_weight` attribute.
```
does this mean this module doesn't have `.weight` present? the language doesn't make that clear, or give the user an understanding of why `.weight` is simply not used and a new attribute name is introduced

zafartahirov replied:
Oh, yea -- this is confusing. The `weight` still exists -- I guess I should remove the `_packed_weight` from the docs.

soumith commented on torch/nn/quantized/functional.py:
```diff
@@ -5,17 +5,61 @@
+      :attr:`inplace` is not supported for the quantized relu.
+
+    Applies the rectified linear unit function element-wise. See
+    :class:`~torch.nn.quantized.ReLU` for more details.
```
again, `torch.nn.functional.ReLU` is what you meant to type here

zafartahirov replied:
Actually, about that -- don't we want to refer the functional to its module counterpart? The documentation in the `torch.nn.quantized.ReLU` is more relevant to this than the non-quantized counterpart

Commit:
Update on "Documentation cleanup"


Differential Revision: [D16414202](https://our.internmc.facebook.com/intern/diff/D16414202)

Commit:
Update on "Documentation cleanup"


Differential Revision: [D16414202](https://our.internmc.facebook.com/intern/diff/D16414202)

Commit:
Update on "Documentation cleanup"


Differential Revision: [D16414202](https://our.internmc.facebook.com/intern/diff/D16414202)

Commit:
Update on "Documentation cleanup"


Differential Revision: [D16414202](https://our.internmc.facebook.com/intern/diff/D16414202)
